posthog/queries/breakdown_props.py

```diff
@@ -233,7 +233,12 @@ def get_breakdown_prop_values(
         query_type="get_breakdown_prop_values",
         filter=filter,
         team_id=team.pk,
-    )[0][0]
+    )
+
+    if filter.using_histogram:
+        return response[0][0]
+    else:
+        return [row[0] for row in response]
```

Comment:
There are two queries here: 1) integer histogram breakdown, 2) string top results breakdown.

The response of the second query changed in order to also return nulls. More on it below.